.github/ISSUE_TEMPLATE/new-metadata-schema-request.yml

```diff
@@ -0,0 +1,53 @@
+name: New Metadata Schema Request
+description: Request a new metadata schema.
+title: "[New Metadata Schema Request]: "
+labels: ["documentation", "enhancement"]
+assignees:
+  - 
+body:
```

Comment:
We are missing a field for schema name or label?

Reply:
done